src/WSSecurity/SignatureCanonicalization.php

```diff
@@ -0,0 +1,17 @@
+<?php
+declare(strict_types=1);
+
+namespace Soap\Psr18WsseMiddleware\WSSecurity;
+
+/**
+ * Currently, there is no option to configure this in robrichards/wsse.
+ * It is hardcoded to C14N
```

Comment:
see https://github.com/robrichards/wse-php/blob/c9611f554d88cbd58935d7da901d84093068168c/src/WSSESoap.php#L221-L227

Reply:
out of scope for now